roles/provision_vm/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Fail if provision_vm_cluster and provision_vm_esxi_hostname both weren't set or both were set
+  when: ( provision_vm_cluster is defined and provision_vm_esxi_hostname is defined ) or ( provision_vm_cluster is not defined and provision_vm_esxi_hostname is not defined) # noqa: yaml[line-length]
```

Comment:
Isn't this handled by the `vmware_guest` module? We don't really need it in case of removal...

Reply:
Maybe for removal I will create a new role? like sean suggested?

Reply:
I will remove that check